src/swsssdk/scripts/sonic-db-cli

```diff
@@ -1,17 +1,43 @@
 #!/usr/bin/python
 import sys
 import swsssdk
+import redis
+import time
+import syslog
 
 argc = len(sys.argv)
-if argc == 2 and sys.argv[1] == '-h':
-    print("""
+if argc == 2:
+    op = sys.argv[1]
+    if op == '-h':
+        print("""
 Example 1: sonic-db-cli CONFIG_DB keys *
 Example 2: sonic-db-cli APPL_DB HGETALL VLAN_TABLE:Vlan10
 Example 3: sonic-db-cli APPL_DB HGET VLAN_TABLE:Vlan10 mtu
 Example 4: sonic-db-cli APPL_DB EVAL "return {KEYS[1],KEYS[2],ARGV[1],ARGV[2]}" 2 k1 k2 v1 v2
+Example 4: sonic-db-cli PING
 """)
-elif argc < 3:
-    msg = "'Usage: sonic-db-cli <db_name> <cmd> [arg [arg ...]]'. See 'sonic-db-cli -h' for detail examples."
+        sys.exit(0)
+    elif op == 'PING':
+        db_insts = swsssdk.SonicDBConfig.get_instance_list()
+        # ping all redis instances one by one
+        for inst in db_insts:
```

Comment:
>for inst in db_insts: [](start = 8, length = 21)

Ping them in parallel?

Reply:
GOOD idea, using multiprocess.Pool to ping all instances together